docs/05-classes/intro.md

```diff
@@ -0,0 +1,70 @@
+---
+nav_order: 1
+title: Clase
+has_children: true
+---
+
+# Clase
+
+În Python, clasele sunt un mod de a organiza și a grupa date și funcționalități într-o unitate logică.
+O clasă este un tip de șablon sau plan pentru crearea obiectelor.
+
+## Cum se definește o clasă?
+
+```python
+class Masina:
+    # Atributele clasei
+    marca = ""
```

Comment:
Ăstea sunt echivalentul unor variabile statice definite pe clasă. Sunt partajate de toate instanțele unei clase: https://docs.python.org/3/tutorial/classes.html#class-and-instance-variables
Ca să fie alocate individual, pt fiecare instanță, trebuie să fie definite doar în constructor